hkusu/slack-post-action
# Update to node20

In this PR:

* Use node20 in action
* Update actions in workflow (to `actions/checkout@v4`)
* Use the latest dependency
  * `@actions/core` to 1.10.1
  * `axios` to 1.6.7
  * `@zeit/ncc` to `@vercel/ncc` 0.38.1
    * Works fine in packaging this action
  * and others
* Change the version to 2.0.0
* Update `dist/index.js`

See https://github.com/k-kojima-yumemi/slack-post-action/actions/runs/7778919010/job/21209277986 for the result of updating dependencies. It seems working fine in my channel ✅
